osu.Game/Screens/Edit/Editor.cs

```diff
@@ -1007,7 +1009,7 @@ private void onModeChanged(ValueChangedEvent<EditorScreenMode> e)
                         throw new InvalidOperationException("Editor menu bar switched to an unsupported mode");
                 }
 
-                LoadComponentAsync(currentScreen, newScreen =>
+                screenContainer.LoadComponentAsync(currentScreen, newScreen =>
```

Comment:
What's this change doing? @smoogipoo looks like one of yours?

I'm not sure it's a big deal but I'm weirded out by it.

Reply:
Previously, the context menu container was stored as a local and cached in this class. Now, the context menu container caches itself via `[Cached]` on the type itself.

But this moves the dependencies one level lower from `Editor` itself, and previously this line would load the screen with the dependencies of the `Editor` rather than the expected lower level.